Gems/ROS2/Code/Source/Lidar/SegmentationClassConfiguration.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/Serialization/EditContextConstants.inl>
+#include <ROS2/Lidar/ClassSegmentationBus.h>
+#include <ROS2/Lidar/SegmentationClassConfiguration.h>
+
+namespace ROS2
+{
+    const SegmentationClassConfiguration SegmentationClassConfiguration::UnknownClass =
+        SegmentationClassConfiguration{ "Unknown", UnknownClassId, AZ::Color(1.0f, 1.0f, 1.0f, 1.0f) };
+    const SegmentationClassConfiguration SegmentationClassConfiguration::GroundClass =
+        SegmentationClassConfiguration{ "Ground", TerrainClassId, AZ::Color(0.5f, 0.25f, 0.0f, 1.0f) };
```

Comment:
Is there any reason behind this color?

Reply:
This color was chosen in the original segmentation pr by @Fireronin